commit 3cd015cce785e48f6e6ea09a09e17f2b1bfbfa90 (refs/changes/50/5250/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2019-01-22T14:21:04-08:00 (9 months ago)
    
    MANTA-4035 retire propeller
    Reviewed by: David Pacheco <dap@joyent.com>

diff --git a/sapi_manifests/madtom_coal/template b/sapi_manifests/madtom_coal/template
index cb3f72c..72baef9 100644
--- a/sapi_manifests/madtom_coal/template
+++ b/sapi_manifests/madtom_coal/template
@@ -93,12 +93,6 @@
         { "processType": "marlin-dashboard", "port": 80 }
       ]
     },
-    {
-      "hostType": "propeller",
-      "processes": [
-        { "processType": "propeller", "port": 80 }
-      ]
-    },
     {
       "hostType": "workflow",
       "processes": [
@@ -151,12 +145,6 @@
       "path": "/index.htm",
       "secure": false
     },
-    {
-      "processType": "propeller",
-      "checkerType": "http",
-      "path": "/ping",
-      "secure": false
-    },
     {
       "processType": "medusa",
       "checkerType": "http",
diff --git a/sapi_manifests/madtom_lab/template b/sapi_manifests/madtom_lab/template
index b77117b..65f567e 100644
--- a/sapi_manifests/madtom_lab/template
+++ b/sapi_manifests/madtom_lab/template
@@ -102,12 +102,6 @@
         { "processType": "marlin-dashboard", "port": 80 }
       ]
     },
-    {
-      "hostType": "propeller",
-      "processes": [
-        { "processType": "propeller", "port": 80 }
-      ]
-    },
     {
       "hostType": "workflow",
       "processes": [
@@ -160,12 +154,6 @@
       "path": "/index.htm",
       "secure": false
     },
-    {
-      "processType": "propeller",
-      "checkerType": "http",
-      "path": "/ping",
-      "secure": false
-    },
     {
       "processType": "medusa",
       "checkerType": "http",
diff --git a/sapi_manifests/madtom_production/template b/sapi_manifests/madtom_production/template
index 4f9a1da..d1b00ae 100644
--- a/sapi_manifests/madtom_production/template
+++ b/sapi_manifests/madtom_production/template
@@ -114,12 +114,6 @@
         { "processType": "marlin-dashboard", "port": 80 }
       ]
     },
-    {
-      "hostType": "propeller",
-      "processes": [
-        { "processType": "propeller", "port": 80 }
-      ]
-    },
     {
       "hostType": "workflow",
       "processes": [
@@ -172,12 +166,6 @@
       "path": "/index.htm",
       "secure": false
     },
-    {
-      "processType": "propeller",
-      "checkerType": "http",
-      "path": "/ping",
-      "secure": false
-    },
     {
       "processType": "medusa",
       "checkerType": "http",
